src/network.c

```diff
@@ -97,3 +101,20 @@ int create_epoll_socket() {
 
     return epoll_descriptor;
 }
+
+int full_message_availiable(int socket) {
+    int message[MAX_SIZE_MESSAGE_INT];
```

Comment:
```suggestion
    uint32_t message[MAX_SIZE_MESSAGE_INT];
```
I think we want to read it as an unsigned int because it doesn't make much sense to have negative messages. Also we define it here:
https://github.com/olincollege/p2p-networking/blob/main/src/message.h#L30